openquake/server/views.py

```diff
@@ -728,7 +736,7 @@ def aristotle_get_rupture_data(request):
             content=json.dumps(response_data), content_type=JSON, status=400)
     rupdic['trts'] = trts
     rupdic['mosaic_models'] = mosaic_models
-    rupdic['rupture_file_from_usgs'] = rupdic['rupture_file']
+    rupdic['rupture_from_usgs'] = rup is not None
```

Comment:
The propagation of the downloaded/converted rupture to the job still needs to be fixed